system-test/test-install.ts

```diff
@@ -74,7 +74,7 @@ const app = express();
 
 app.get('/error', (req, res, next) => {
   res.send('Something broke!');
-  next(new Error('Custom error message'));
+  next!(new Error('Custom error message'));
```

Comment:
Seems fine because this is a test but why can't this be
```ts
if (next) {
  next(new Error...);
}
```
?

Reply:
It could be either way.  The code is not actually run.  It is used to test that the types match what is expected of express.